### ISSUE TITLE
Update locustio/locust Docker image to Python 3.9.6

### ISSUE THREAD

**Issue by istepanko** [locustio/locust#1821]
Python 3.10 will be released in 2 months, ideally package should not be more than 1 release cycle behind.

### Describe the solution you'd like
Update Python version to 3.9.6

### Additional context
I am using Locust + Python 3.9.x for already 6 months and 3.9.6 for a month - no new bugs found.

**Comment by cyberw** [comment]
I dont think we should upgrade just yet. 3.8 is still the most common version in the wild, and 3.9 hasnt really taken off yet (only ~6% of locust pypi downloads are 3.9, both 3.7 and 3.8 are much more common). Maybe in 6 months or so.

**Comment by StevenBorg** [comment]
One note, that may be relevant. running Snyk against the latest locustio/locust docker instance, shows 361 vulnerabilities.

Repro steps
`docker pull locustio/locust:latest`
`docker scan --accept-license locustio/locust`

Notes from scan and recommended steps (not including the long list of security vulnerabilities):
Package manager:   deb
Project name:      docker-image|locustio/locust
Docker image:      locustio/locust
Platform:          linux/amd64
Base image:        python:3.8.11-buster

Tested 431 dependencies for known vulnerabilities, found 361 vulnerabilities.

Your base image is out of date
1) Pull the latest version of your base image by running 'docker pull python:3.8.11-buster'
2) Rebuild your local image

For more free scans that keep your images secure, sign up to Snyk at https://dockr.ly/3ePqVcp

**Comment by cyberw** [comment]
I think these are separate issues, but I'm starting to think 3.9 is a good idea now (it has been a while longer, and I realized that it doesnt really matter, because probably nobody needs it to be running specifically 3.8).

3.8 is always _latest_ 3.8.x, so it should be the same though? And how did you end up with python:3.8.11-*buster*?

**Comment by cyberw** [comment]
Anyway, PR for 3.9 is welcome.
